tools/make/kube.mk

```diff
@@ -79,6 +79,7 @@ kube-deploy-for-benchmark-test: manifests helm-generate ## Install Envoy Gateway
 	helm install eg charts/gateway-helm --set deployment.envoyGateway.imagePullPolicy=$(IMAGE_PULL_POLICY) \
 		--set deployment.envoyGateway.resources.limits.cpu=$(BENCHMARK_CPU_LIMITS) \
 		--set deployment.envoyGateway.resources.limits.memory=$(BENCHMARK_MEMORY_LIMITS) \
+		--set config.envoyGateway.admin.enablePprof=true \
```

Comment:
curious if setting this know affected existing benchmark throughput ?

Reply:
compare two benchmark results: https://github.com/envoyproxy/gateway/actions/runs/10163594751/job/28107439647?pr=3951 (added) & https://github.com/envoyproxy/gateway/actions/runs/10097739129/job/27970475986?pr=3682 (not added), the results almost the same.